kiwi/system/uri.py

```diff
@@ -210,15 +219,29 @@ def _obs_distribution(self, name):
             )
         return obs_distribution
 
-    def _suse_buildservice_path(self, name):
+    def _suse_buildservice_path(self, name, urischeme, fragment=None):
         """
         Special to openSUSE buildservice. If the buildservice builds
         the image it arranges the repos for each build in a special
         environment, the so called build worker.
         """
-        return self._local_path(
-            '/usr/src/packages/SOURCES/repos/' + name
-        )
+        bs_source_dir = '/usr/src/packages/SOURCES'
+        if self.repo_type == 'container':
+            if urischeme == 'obsrepositories':
+                local_path = os.sep.join(
+                    [bs_source_dir, 'containers/_obsrepositories', name]
+                )
+            else:
+                local_path = os.sep.join(
+                    [bs_source_dir, 'containers', name]
+                )
+            if fragment:
+                local_path = ''.join([local_path, '#', fragment])
```

Comment:
All looks great to me :+1 I am just wondering if OBS backend is going to be the responsible of putting in place the appropriate file with the exact name (e.g. `/usr/src/packages/SOURCES/containers/opensuse`) or it is KIWI going to use some kind of regexp match on any file inside the specified folder (e.g. first match of `opensuse*.docker.tar.xz` could be something like `opensuse.x86_64-1.0.0.docker.tar.xz`).